packages/tab/src/react/__stories__/index.story.js

```diff
@@ -1,8 +1,8 @@
-import PropTypes from 'prop-types'
-import React from 'react'
+import * as PropTypes from 'prop-types'
```

Comment:
Does this change avoid having to set es module compatibility with TS?  If so, me likey.

Reply:
Maybe? I just know that TS seems to be stricter in ES standards and it wanted it this way when running through `tsc`. Storybook still worked OK as well.